src/index.js

```diff
@@ -501,6 +526,55 @@ const initialize = async () => {
       }
     };
 
+    /**
+     * Multisig
+     */
+
+    deployMultisigButton.onclick = async () => {
+      multisigContractStatus.innerHTML = 'Deploying';
+
+      try {
+        multisigContract = await multisigFactory.deploy();
+        await multisigContract.deployTransaction.wait();
+      } catch (error) {
+        multisigContractStatus.innerHTML = 'Deployment Failed';
+        throw error;
+      }
+
+      if (multisigContract.address === undefined) {
+        return;
+      }
+
+      console.log(
+        `Contract mined! address: ${multisigContract.address} transactionHash: ${multisigContract.deployTransaction.hash}`,
+      );
+      multisigContractStatus.innerHTML = 'Deployed';
+      sendMultisigButton.disabled = false;
+    };
+
+    sendMultisigButton.onclick = async () => {
+      try {
+        const result = await ethereum.request({
+          method: 'eth_sendTransaction',
+          params: [
+            {
+              from: accounts[0],
+              to: multisigContract.address,
+              value: '0x16345785D8A0', // 24414062500000
```

Comment:
This looks great!
Small suggestion: should we update the value to `0x0`?
See: [#129](https://github.com/MetaMask/test-dapp/pull/129)

Reply:
Thanks for pointing this @PeterYinusa . In this case, we are not sending ETH to an existing random or hacked address but to the contract address that we just deployed on the previous step.
Notice the same pattern for Deposit.

![image](https://user-images.githubusercontent.com/54408225/217184075-2b2c1f65-8667-4b53-ab7b-1a244ab95bdd.png)

I opted for adding a small value, as I think testing with a 0 Send value might make that we miss some issues. So I really wanted to make sure that we send some value. 

What do you think?